frontend/src/Components/Glyphs.js

```diff
@@ -0,0 +1,142 @@
+import React, { Component } from "react";
+import * as d3 from "d3";
+import "./Glyphs.css";
+
+const availableColor = '#0D77AC';
+const availableColorLighter = '#C1D6E4';
+const unavailableColor = '#d8d8d8';
+const bestPractices = [
+		'no_fee',
+		'no_notary_required',
+		'no_witness_required',
+		'office_contact',
+		'electronic_request'
+	];
+
+class Glyphs extends Component {
+
+	constructor(props) {
+        super(props);
+        this.state = {
+            states:props.states
+        };
+    }
+
+	componentDidMount() {
+        this.initVis();
+    }
+
+    initVis() {
+        var vis = this;
+
+        vis.margin = { top: 20, right: 60, bottom: 200, left: 60 };
+
+        vis.width = 1000 - vis.margin.left - vis.margin.right;
+        vis.height = 900 - vis.margin.top - vis.margin.bottom;
+
+        vis.svg = d3.select(".scorecard").append("div")
+            .attr("width", vis.width + vis.margin.left + vis.margin.right)
+            .attr("height", vis.height + vis.margin.top + vis.margin.bottom)
+            .attr("class", "scorecard-container")
+            .attr("transform", "translate(" + vis.margin.left + "," + vis.margin.top + ")");
+
+        vis.drawGlyphs();
+    }
+
+    drawSingleGlyph(state, g) {
+
+    	const practicesAvailble = bestPractices.filter(p => state[p] === true);
+    	const allPractices = bestPractices.filter(p => state[p] === true || state[p] === false);
+
+    	const width = 80;
+    	const height = 80;
+		const center = { x: width / 2, y: height / 2 };
+		const glyphRadius = Math.round(height / 3);
+		const circleRadius = Math.round(glyphRadius / 4);
+		function pentagonVertex(i) {
+		    const angle = i / 5 * Math.PI * 2 + 1.1 * Math.PI;
+		    return {
+		    	x: Math.cos(angle) * glyphRadius + center.x,
+		    	y: Math.sin(angle) * glyphRadius + center.y
+		    };
+	    }
+
+	    const integration = g.selectAll('.integration')
+		    .data(practicesAvailble)
+		    .enter().append('g')
+		    .attr('class', 'integration');
+
+		integration.append('path')
+		    .attr('d', d => {
+		    	const points = []
+		    	var i = 0;
+		    	for (var key in state) {
+				    if (state[key] === true) {
```

Comment:
maybe reduce indentation?

Reply:
Hmm it doesn't appear to have this much indentation for me in Sublime, not sure what's causing this